src/schedule-and-details/pacing-section/PacingSection.scss

```diff
@@ -0,0 +1,15 @@
+.pacing-section {
+  label.pgn__form-label {
+    @extend %heading-h6;
+  }
+
+  input.pgn__form-radio-input {
+    padding: .5rem;
+    height: 1rem;
+    width: 1rem;
```

Comment:
`1rem` its `$spacer` variable from Paragon. Please add below and in other files that you have created.

Reply:
changed